diem-unified-charter.md

```diff
@@ -32,8 +33,8 @@ Which attributes an emblem contains, and how a digital emblem is secured and pre
 
 # Initial Scope
 
-The DIEM WG will ensure maximum reuse of work for emblem representation and binding. 
-A discovery mechanism for the initial work will only be specified by this group after the initial emblem binding protocol is completed (see the Deliverables section below).
+The DIEM WG will ensure maximum reuse of work for emblem representation and validation.
+A discovery mechanism for the initial work will only be specified by this group after the initial emblem validation procedure is completed (see the Deliverables section below).
```

Comment:
I realize that you didn't change this sentence, but I would question if you can address validation, without considering the discovery mechanism... If you can, this implies that emblems are secured in a standalone way, and do not rely on "channel binding"

Reply:
I think this connects to #33. The sentence is problematic for many reasons. Maybe this change illustrates these problems. My proposal would be to merge this PR (provided we have consensus) and fix this sentence separately.